test-poc/framework/src/main/java/org/keycloak/test/framework/server/DistributionKeycloakTestServer.java

```diff
@@ -1,9 +1,11 @@
 package org.keycloak.test.framework.server;
 
+import org.keycloak.test.framework.database.TestDatabase;
+
 public class DistributionKeycloakTestServer implements KeycloakTestServer {
 
     @Override
-    public void start(KeycloakTestServerConfig serverConfig) {
+    public void start(KeycloakTestServerConfig serverConfig, TestDatabase testDatabase) {
```

Comment:
Maybe it would be cleaner to just pass the DatabaseConfig? Makes it clear that it shouldn't be stopping/starting the db or anything like that ;)